BinaryDecompiler/internal_includes/structs.h

```diff
@@ -5,8 +5,8 @@
 #include <string>
 #include "hlslcc.h"
 
-#include "internal_includes/tokens.h"
-#include "internal_includes/reflect.h"
+#include "tokens.h"
+#include "reflect.h"
```

Comment:
The BinaryDecompiler project is a C++ port of https://github.com/James-Jones/HLSLCrossCompiler

Please refrain from making any unecessary changes in this third party code - we should be able to diff our code with upstream and see exactly what we have changed.